Source/SwiftLintFramework/Rules/ForceUnwrappingRule.swift

```diff
@@ -32,7 +33,7 @@ public struct ForceUnwrappingRule: OptInRule, ConfigurationProviderRule {
     )
 
     public func validateFile(file: File) -> [StyleViolation] {
-        return file.matchPattern("[\\w)]+!", withSyntaxKinds: [.Identifier]).map {
+        return file.matchPattern("([\\w)]+!)|(\"\\)!)", withSyntaxKinds: [.Identifier, .String]).map {
```

Comment:
I think this will do it: `"([\\w\"])+([\\s]*[)]+)*!"`
- Match word or `"` 1 or more times
- (Optional whitespace match + One or more `)` ) Zero or more times
- Match `!`

http://www.regexpal.com/?fam=94142